src/index.js

```diff
@@ -16,6 +21,10 @@ root.render(
       <Route exact path='/Login'element={<Login/>}/> 
       <Route exact path='/register'element={<Register/>}/> 
       <Route exact path='/ForgotPassword'element={<ForgotPassword/>}/> 
+      <Route exact path='/ResetPassword'element={<ResetPassword/>}/> 
+      <Route exact path='/'element={<Cart/>}/>
```

Comment:
Please delete line 25 in index.js
<Route exact path='/'element={<Cart/>}/>

"/" will be used for the home path